### ISSUE TITLE
why need the tf.nn.l2_loss?

### ISSUE THREAD

**Issue by 1453042287** [HiKapok/SSD.TensorFlow#58]
like the title

**Comment by HiKapok** [comment]
@1453042287 l2_loss is used for weight decay of params
